app/views/jobs/_form.html.erb

```diff
@@ -1,15 +1,34 @@
-<%= form_for [@company, @job] do |f| %>
-  <%= f.label :title %>
-  <%= f.text_field :title %>
+<%= form_for [@company, @job], :html => {:class => "form-horizontal center"} do |f| %>
+  <%= render partial: "shared/errors", locals: { target: @job } %>
 
-  <%= f.label :description %>
-  <%= f.text_field :description %>
+  <div class="form-group">
+    <%= f.label :title, class: "control-label" %>
+    <%= f.text_field :title, class: "form-control", placeholder: "Job Title" %>
+  </div>
 
-  <%= f.label :level_of_interest %>
-  <%= f.number_field :level_of_interest %>
+  <div class="form-group">
+    <%= f.label :description, class: "control-label" %>
+    <%= f.text_field :description, class: "form-control", placeholder: "Job Description" %>
+  </div>
 
-  <%= f.label :city %>
-  <%= f.text_field :city %>
+  <div class="form-group">
+    <%= f.label :level_of_interest, class: "control-label" %>
+    <%= f.number_field :level_of_interest, class: "form-control", placeholder: "0 - 100  (0 is low, 100 is high)" %>
+  </div>
+
+  <div class="form-group">
+    <%= f.label :city, class: "control-label" %>
+    <%= f.text_field :city, class: "form-control", placeholder: "City" %>
+  </div>
+
+  <div class="form-group" style="margin-bottom: 0px;">
+    <%= f.label "Category", class: "control-label" %>
+    <%= f.collection_select :category_id, Category.all, :id, :title, :prompt => 'Select Category' %>
```

Comment:
this is too much logic in your view. what part of MVC is responsible for querying our database to prep data for a view?